cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -377,7 +377,7 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 	osbuildJobResult.OSBuildOutput, err = osbuild.RunOSBuild(jobArgs.Manifest, impl.Store, outputDirectory, exports, nil, extraEnv, true, os.Stderr)
 	// First handle the case when "running" osbuild failed
 	if err != nil {
-		osbuildJobResult.JobError = clienterrors.WorkerClientError(clienterrors.ErrorBuildJob, "osbuild build failed", nil)
+		osbuildJobResult.JobError = clienterrors.WorkerClientError(clienterrors.ErrorBuildJob, "running osbuild build failed", nil)
```

Comment:
not sure if this change is actually useful :thinking:

Reply:
there is the same error message below in the same function, I simply want to distinguish where it was failed

Reply:
Would setting the error detail instead work for you (the `nil` value)?